### ISSUE TITLE
about train epoch

### ISSUE THREAD

**Issue by QuiPyJu** [infly-ai/INF-MLLM#17]
First of all, thank you very much for your work. It is stated in the paper that all data were trained for 1 epoch. Why not train for more epochs? Did the metrics decrease?

**Comment by QuiPyJu** [comment]
By the way, is there any plan to open-source the training dataset?

**Comment by weizhenli012** [comment]
In our preliminary experiments, the reward rapidly increased to over 0.9 (out of a total of 1) within a single epoch, indicating that the training samples were already simple for the model. Additionally, we also tried increasing the number of epochs but observed no extra benefits. Therefore, we kept the training to one epoch for all subsequent experiments.

**Comment by weizhenli012** [comment]
> By the way, is there any plan to open-source the training dataset?

It will be released soon.

**Comment by OhmaAllen** [comment]
请问你们这里说的epoch指的是给整个数据集重复的次数吗，我用的openrlhf，这里的“episodes”更像是传统的“epoch”，这里的epoch指的好像是对于同一批次小数据重复的次数。比如数据总共分3批为1，2，3.  如果是5个episode就是，123，123，123，123，123重复五次。但是5个epoch就是11111，22222，33333每一批分别重复五次。所以我想问一下你们的episode设的是几呢，还是这个epoch指的就是episode

**Comment by weizhenli012** [comment]
> 请问你们这里说的epoch指的是给整个数据集重复的次数吗，我用的openrlhf，这里的“episodes”更像是传统的“epoch”，这里的epoch指的好像是对于同一批次小数据重复的次数。比如数据总共分3批为1，2，3. 如果是5个episode就是，123，123，123，123，123重复五次。但是5个epoch就是11111，22222，33333每一批分别重复五次。所以我想问一下你们的episode设的是几呢，还是这个epoch指的就是episode

Here the epoch means the episode you say.
